department-of-veterans-affairs/caseflow
# Fix for Package document type

Resolves [Related Correspondence Package Document Type not showing in Intake](https://jira.devops.va.gov/browse/APPEALS-52549)

# Description
Please explain the changes you made here.

## Acceptance Criteria

## Testing Plan


1. Log in as INBOUND_OPS_TEAM_MAIL_INTAKE_USER
2. Navigate to Queue
3. Click the Switch Views dropdown and select Your Correspondence
4. Click a correspondence from the table
5. On the Review Package page, select the correspondence type and click save. 
6. Click create record at the bottom of the Review Package form.
7. On intake step 1, click the radio button to link prior correspondence.
8. Verify that the Package Document Type should show in the table


# Frontend
## User Facing Changes

 BEFORE|AFTER
 ---|---

## Storybook Story
*For Frontend (Presentation) Components*

# Backend
## Database Changes
*Only for Schema Changes*


## Integrations: Adding endpoints for external APIs

# Best practices
## Code Documentation Updates

## Tests
### Test Coverage
Did you include any test coverage for your code? Check below:

### Code Climate
Your code does not add any new code climate offenses? If so why?

## Monitoring, Logging, Auditing, Error, and Exception Handling Checklist

### Monitoring

### Logging

### Auditing

### Error Handling

### Exception Handling
